@navikt/core/react/src/accordion/AccordionHeader.tsx

```diff
@@ -41,11 +51,7 @@ const AccordionHeader = forwardRef<HTMLButtonElement, AccordionHeaderProps>(
             aria-hidden
           />
         </span>
-        <Heading
-          size={accordionContext?.headingSize ?? "small"}
-          as="span"
-          className="navds-accordion__header-content"
```

Comment:
The classname is referred to in several repos. Should we wait until next major before removing it? Should at least add it to deprecated classes list and create changeset.

Reply:
Whoops, did a search and found no hits, but think i searched wrong org or something. Re-added it for now 👍